workflow-templates/hybrid-ci.yml

```
@@ -148,11 +189,11 @@ jobs:
       ######         🧼 POST-BUILD CLEANUP 🧹          ######
       #######################################################
       - name: 🧹 Post Build Cleanup 🧹
-        if: always()
+        if: ${{ needs.Path-Filtering.outputs.non-markdown-changed == 'true' }} &&
```

```suggestion
        if: ${{ needs.Path-Filtering.outputs.non-markdown-changed == 'true' }}
```
Shouldn't need these

---

Or maybe you wanted the `&& always()`

---

good catch, I have no idea where that disappeared to lol